packages/core/src/payment/strategies/squarev2/squarev2-script-loader.ts

```diff
@@ -0,0 +1,25 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { assertSquareV2Window } from './is-squarev2-window';
+import { Square } from './types';
+
+export enum SquareV2WebPaymentsSdkEnv {
+    LIVE = 'https://web.squarecdn.com/v1/square.js',
+    SANDBOX = 'https://sandbox.web.squarecdn.com/v1/square.js',
+}
+
+export default class SquareV2ScriptLoader {
+    constructor(private _scriptLoader: ScriptLoader) {}
+
+    async load(testMode = false): Promise<Square> {
+        await this._scriptLoader.loadScript(
+            testMode
+                ? SquareV2WebPaymentsSdkEnv.SANDBOX
+                : SquareV2WebPaymentsSdkEnv.LIVE
```

Comment:
Perhaps we don't need to keep the test settings in the production code.

Reply:
Sorry, I'm not sure about this question. Could you elaborate more? 😬

Reply:
My bad. I didn't reallise it could be a test mode of `Square` for merchants. If so, then all good.